desktop/ui/pom.xml

```diff
@@ -182,6 +187,75 @@
 			<groupId>org.freemarker</groupId>
 			<artifactId>freemarker</artifactId>
 		</dependency>
+
+		<!-- Engough Hadoop/Spark for uploading and submitting jobs -->
+		<dependency>
+			<groupId>org.apache.spark</groupId>
+			<artifactId>spark-launcher_2.11</artifactId>
+			<version>${spark.version}</version>
```

Comment:
I guess we should have this dependency in dependencyManagement so that you don't need to specify the version or the exclusions